src/Raven.Studio/typescript/models/database/tasks/ongoingTaskBackupListModel.ts

```diff
@@ -62,8 +62,17 @@ class ongoingTaskBackupListModel extends ongoingTaskListModel {
     initializeObservables() {
         super.initializeObservables();
 
-        const urls = appUrl.forCurrentDatabase();
-        this.editUrl = urls.editPeriodicBackupTask(this.taskId);
+        this.isServerWide = ko.pureComputed(() => {
+            return this.taskName().startsWith(ongoingTaskBackupListModel.serverWideNamePrefixInStudio);
+        });
+        
+        this.editUrl = ko.pureComputed(()=> {
+             const urls = appUrl.forCurrentDatabase();
+             
+             return this.isServerWide() ?  appUrl.forEditServerWideBackup(this.taskName().replace(ongoingTaskBackupListModel.serverWideNamePrefixInStudio +', ', '')) :
```

Comment:
can we be consistent with server?

Reply:
As discussed:
1. Ongoing-tasks-list-view will show: `SERVER WIDE BACKUP, <TASK NAME> `
2. Server-Wide-backups-list-view will show just:  `<TASK NAME>`